### PR TITLE
Issue 2: Save Image Link

### PR DESCRIPTION
**Pull request by hannaveldhuizen**
We created endpoints to create a new image in the database and display it to the user. We also added a feature to validate the URL given by the user.

### CODE REVIEW

**Comment by hannaveldhuizen** [comment]
[Issue 2: Save Image Link](https://trello.com/c/MPTFPT3B/139-issue-2-save-image-link)

File: app/views/images/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Create New Image</h1>
+<%= simple_form_for @image do |f| %>
+<%= f.input :url %>
```
Comment (eldonchen0501) [inline app/views/images/new.html.erb]: We might need indent here.

File: test/controllers/images_controller_test.rb [hunk 2]
```diff
@@ -11,4 +11,16 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
     assert_select 'h1', 'Create New Image'
     assert_select '#image_url', count: 1
   end
+
+  test 'should create image' do
+    assert_difference('Image.count', 1) do
```
Comment (eldonchen0501) [inline test/controllers/images_controller_test.rb]: we can get rid of the parentheses here.
`assert_difference 'Image.count', 1 do`

File: test/controllers/images_controller_test.rb [hunk 3]
```diff
@@ -11,4 +11,16 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
+      post images_url, params: { image: { url: 'https://hello.com' } }
+    end
+  end
+
+  test 'should not create an image with an invalid url' do
+    assert_no_difference('Image.count') do
```
Comment (eldonchen0501) [inline test/controllers/images_controller_test.rb]: same here

**Comment by fanniey** [comment]
👍 Nicely done.

**Pull request merged**